desc/vmec.py

```diff
@@ -814,10 +814,10 @@ def save(cls, eq, path, surfs=128, verbose=1):
             x_mn[:, :] = full_transform.fit(data.T).T
         xm, xn, s, c = ptolemy_identity_rev(m, n, x_mn)
         bsubvmnc[0, :] = 0
-        bsubvmnc[1:, :] = c * signgs[:]
+        bsubvmnc[1:, :] = c
```

Comment:
So `B^theta` and `B_theta` still need the sign corrections, but `B^zeta` and `B_zeta` do not? This seems odd to me.